_topic_maps/_topic_map_rosa.yml

```diff
@@ -365,6 +355,75 @@ Topics:
 # - Name: Using the internal registry
 #   File: rosa-using-internal-registry
 ---
+Name: Logging
+Dir: logging
+Distros: openshift-rosa
+Topics:
+- Name: Release notes
+  File: cluster-logging-release-notes
+- Name: About Logging
+  File: cluster-logging
+- Name: Installing Logging
+  File: cluster-logging-deploying
+- Name: Accessing the service logs
+  File: sd-accessing-the-service-logs
+- Name: Viewing cluster logs in the AWS Console
+  File: rosa-viewing-logs
+- Name: Configuring your Logging deployment
+  Dir: config
+  Topics:
+  - Name: About the Cluster Logging custom resource
+    File: cluster-logging-configuring-cr
+  - Name: Configuring the logging collector
+    File: cluster-logging-collector
+  - Name: Configuring the log store
+    File: cluster-logging-log-store
+  - Name: Configuring the log visualizer
+    File: cluster-logging-visualizer
+  - Name: Configuring Logging storage
+    File: cluster-logging-storage-considerations
+  - Name: Configuring CPU and memory limits for Logging components
+    File: cluster-logging-memory
+  - Name: Using tolerations to control Logging pod placement
+    File: cluster-logging-tolerations
+  - Name: Configuring systemd-journald and Fluentd
+    File: cluster-logging-systemd
+  - Name: Maintenance and support
+    File: cluster-logging-maintenance-support
+- Name: Logging with the LokiStack
+  File: cluster-logging-loki
+- Name: Viewing logs for a specific resource
+  File: viewing-resource-logs
+- Name: Viewing cluster logs in Kibana
+  File: cluster-logging-visualizer
+- Name: Forwarding logs to third party systems
+  File: cluster-logging-external
+- Name: Enabling JSON logging
```

Comment:
I didn't review items that were included in the topic maps but unchanged. If you didn't already, I'd just go through everything to make sure you don't have any sentences that don't read right with {product-title} (like saying "an Red Hat", etc.